lib/utils/date.dart

```diff
@@ -23,7 +24,7 @@ class BreezDateUtils {
 
   static String formatTimelineRelative(DateTime d) {
     if (DateTime.now().subtract(Duration(days: 4)).isBefore(d)) {
-      return timeago.format(d, locale: Platform.localeName);
+      return timeago.format(d, locale: getSystemLocale().languageCode);
```

Comment:
Using just the language code to avoid default "English" translations if the locale doesn't match language and code.

e.g:
es_MX should use `es` instead of `en`.
pt_AO should use `pt` instead of `en`.